components/ui/Title.jsx

```diff
@@ -1,7 +1,7 @@
 import {H2} from '../ui/Headings'
 const Title = ({content, styles}) => {
     return (
-        <div className={`bg-center text-white  bg-hero-pattern bg-no-repeat max-w-max ${styles}`}>
+        <div className={`bg-center text-white bg-hero-pattern bg-no-repeat max-w-max ${styles}`}>
             <H2 text={content} styles='pb-11.75 '></H2>
```

Comment:
The space in the class names passed into `styles` prop should be removed.

Reply:
thanks for pointing out, it's resolved!!